common/requirements.txt

```diff
@@ -1 +0,0 @@
-git+https://github.com/eclipse-velocitas/vehicle-model-generator.git@v0.4.0
```

Comment:
Shall we keep this file at all?

Reply:
I left it, maybe we will have some other dependencies soon, would need to change all dockerfiles to..